wurst/objects/items/ScrollsDefinition.wurst

```diff
@@ -0,0 +1,421 @@
+
+@compiletime function createFireBallScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_FIREBALL)
+        ..setCooldownGroup(ABILITY_SPEAR.toRawCode())
+        ..setLumberCost(7)
+        ..setAbilities(ABILITY_FIREBOLT.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollRed)
+        ..setInterfaceIcon(Icons.bTNScrollOfHealing)
+        ..setNameEnhance("Scroll of Fire Ball")
+        ..setTooltipExtended(FB_TT)
+
+@compiletime function createLivingDeadScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_LIVING_DEAD)
+        ..setCooldownGroup(ABILITY_SKELETAL_MINION.toRawCode())
+        ..setLumberCost(7)
+        ..setAbilities(ABILITY_SKELETAL_MINION.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollYellow)
+        ..setInterfaceIcon(Icons.bTNSnazzyScroll)
+        ..setNameEnhance("Scroll of Living Dead")
+        ..setTooltipExtended(LD_TT)
+
+@compiletime function createStoneShieldScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_STONE_ARMOR)
+        ..setCooldownGroup(ABILITY_FROST_ARMOR.toRawCode())
+        ..setLumberCost(9)
+        ..setAbilities(ABILITY_FROST_ARMOR.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollOrange)
+        ..setInterfaceIcon(Icons.bTNScrollUber)
+        ..setNameEnhance("Scroll of Stone Shield")
+        ..setTooltipExtended(SS_TT)
+
+@compiletime function createHasteScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_HASTE)
+        ..setCooldownGroup(ABILITY_HASTE_AOE.toRawCode())
+        ..setLumberCost(10)
+        ..setAbilities(ABILITY_HASTE_AOE.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollHaste)
+        ..setInterfaceIcon(Icons.bTNScrollOfHaste)
+        ..setNameEnhance("Scroll of Haste")
+        ..setTooltipExtended(SH_TT)
+
+@compiletime function createTsunamiScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_TSUNAMI)
+        ..setCooldownGroup(ABILITY_TSUNAMI.toRawCode())
+        ..setLumberCost(9)
+        ..setAbilities(ABILITY_TSUNAMI.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollPurple)
+        ..setInterfaceIcon(Icons.bTNSnazzyScrollPurple)
+        ..setNameEnhance("Scroll of Tsunami")
+        ..setTooltipExtended(ST_TT)
+
+@compiletime function createCycloneScroll() returns ItemDefinition
+    return createUsableItem(ITEM_SCROLL_CYCLONE)
+        ..setCooldownGroup(ABILITY_CYCLONE.toRawCode())
+        ..setLumberCost(9)
+        ..setAbilities(ABILITY_CYCLONE.toRawCode())
+        ..setStockReplenishInterval(120)
+        ..setModelUsed(LocalItems.scrollCyan)
+        ..setInterfaceIcon(Icons.bTNBansheeMaster)
+        ..setNameEnhance("Scroll of Cyclone")
+        ..setTooltipExtended(SC_TT)
+
+
+let BUFF_CYCLONE = 'Bcyc'
```

Comment:
Can you leave a comment explaining why both of these are needed?

Reply:
Updated.